addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -22,30 +22,31 @@
     </template>
 
     <template id="product_cart_lines" inherit_id="website_sale.cart_lines" active="True">
-        <div name="o_wsale_cart_line_button_container" position="inside">
-            <span
-                class="d-none d-md-inline-block ms-1"
+        <xpath expr="//div[@name='o_wsale_cart_line_button_container']" position="inside">
+            <a
+                t-if="line._is_sellable() and line.product_id.type != 'combo'"
```

Comment:
It just come to my attention that in master, you can add a combo product to the Wishlist. Why do you remove this possibility in your PR?
```suggestion
                t-if="line._is_sellable()"
```

Reply:
Sorry it's a mistake, I don't see any reasons why I did that so I probably messed it up during a rebase or I forgot to adapt after making some tests 🤔 Thanks for catching it!